src/V2/Actor/Writer.service.yml

```diff
@@ -1,7 +1,11 @@
+parameters:
+  Neighborhoods_Buphalo_V2_Actor_WriterInterface_IgnoreSourceDirectoryFilesDefault: false
+  Neighborhoods\Buphalo\V2\Actor\WriterInterface.IgnoreSourceDirectoryFiles: '%env(bool:default:Neighborhoods_Buphalo_V2_Actor_WriterInterface_IgnoreSourceDirectoryFilesDefault:Neighborhoods_Buphalo_V2_Actor_WriterInterface__IgnoreSourceDirectoryFiles)%'
```

Comment:
Yeah, I'm not a huge fan of it, but couldn't think of a better way to have it use strong type-casing on the actual property type. Closest I could get would be making it something more like `setWriteMode(string $writeMode)` which supported a couple values (e.g. `ignore_source_directory` and `skip_if_file_exists_in_src`), but that felt overkill because we'd need to add a bunch of other stuff. Ideally I'd only really want to go that route if we could think of other write modes AND it'd be nice if we could use something like an Enum `WriteMode::from(...)` call somewhere in the symfony calling stack and have `setWriteMode(WriteMode $writeMode)`, but that would also only work on PHP 8.1+